pcdscalc/be_lens_calcs.py

```diff
@@ -0,0 +1,830 @@
+'''
+Module for Beryllium Lens Calculations
+'''
+import logging
+import os
+import shutil
+from datetime import date
+from itertools import product
+
+import numpy as np
+import xraydb as xdb
+
+
+logger = logging.getLogger(__name__)
+
+# Set of Be lenses with thicknesses:
+LENS_RADII = [50e-6, 100e-6, 200e-6, 300e-6, 500e-6, 1000e-6, 1500e-6]
+
+FWHM_SIGMA_CONVERSION = 2.35482004503
+WAVELENGTH_PHOTON = 1.2398
+
+
+def photon_to_wavelength(energy):
+    '''
+    Find the wavelength in micrometers, using the
```

Comment:
Docstrings look good, but oddly formatted. Try setting your editor to wrap at 79 chars and reformat the block as appropriate.

In Vim, for example, can do this easily when you do a visual selection of these lines and type `gw` in normal mode. I'm sure if you're using an IDE there's something similar.

Reply:
I ended up using Vim to correct some of the docstrings, because my IDE (Visual Studio Code) would only show what word I should wrap at, but not do it automatically - I need to investigate my IDE a bit more though to see if I have this option. 

I still don't know for sure if the docstrings look fine now - please let me know.

Reply:
Much better!